### PR TITLE
[11.0][MIG] storage_backend

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+keychain https://github.com/OCA/server-auth 11.0
+component https://github.com/OCA/connector 11.0
```
Comment (max3903) [inline oca_dependencies.txt]: @bodedra Just put the repository name.

```
server-auth
connector
```

**Review by max3903 — CHANGES_REQUESTED** [review]

**Comment by max3903** [comment]
:+1:

**Comment by guewen** [comment]
@sebastienbeau we are using this PR and #42 with success (not yet in production though)

**Comment by sebastienbeau** [comment]
great news ! really happy for the migration and to see other people using it ! I will review it this week !

**Review by max3903 — APPROVED** [review]

**Comment by florian-dacosta** [comment]
@bodedra Thanks for the migration
I will close this as we are going to port the modules to version 12, and then backport it to towards version 11.

**Pull request closed without merging**